package.json

```diff
@@ -42,7 +42,7 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "0.66.x",
+        "branch": "c75bump-0.66.x",
```

Comment:
why the branch change here?

Reply:
This was for CI. Removed the commit with this changed.

Reply:
explicit pinning won't be needed anymore as I'll uplift my pipeline changes today